common/components/UI/DomainsComponents/DomainModal/DomainForm/index.tsx

```diff
@@ -14,26 +14,13 @@ interface Props {
 const DomainForm: FC<Props> = ({ form, currentDomain }) => {
   const initialValues = useInitialValues(currentDomain)
   return (
-    <Form
-      form={form}
-      layout="vertical"
-      className={s.Form}
-      initialValues={initialValues}
-    >
+    <Form form={form} layout="vertical" className={s.Form} initialValues={initialValues}>
       <Form.Item name="name" label="Назва" rules={validateField('description')}>
-        <Input
-          placeholder="Вкажіть значення"
-          maxLength={256}
-          className={s.formInput}
-        />
+        <Input placeholder="Вкажіть значення" maxLength={256} className={s.formInput} />
       </Form.Item>
       <EmailSelect form={form} />
-      <DomainStreets />
-      <Form.Item
-        name="description"
-        label="Опис"
-        rules={validateField('required')}
-      >
+      <FormStreetSelect form={form} selectProps={{ multiple: true }} />
```

Comment:
If this approach is working correctly you can use `selectProps={{ mode: 'multiple' }}` and remove all additional typing for `StreetSelect` (like `multiple?: boolean` and it`s usage)